perfume-api/src/main/java/io/perfume/api/sample/application/port/SampleQueryRepository.java

```diff
@@ -0,0 +1,13 @@
+package io.perfume.api.sample.application.port;
+
+import io.perfume.api.sample.domain.Sample;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface SampleQueryRepository {
+
+    List<Sample> find();
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 4, expected level should be 2.